overrides/config/alexsmobs.toml

```diff
@@ -611,22 +611,22 @@
 		farseerSpawnRolls = 0
 		#Spawn Weight, added to a pool of other mobs for each biome. Higher number = higher chance of spawning. 0 = disable spawn
 		#Range: 0 ~ 1000
-		skreecherSpawnWeight = 10
+		skreecherSpawnWeight = 40
 		#Random roll chance to enable mob spawning. Higher number = lower chance of spawning
 		#Range: > 0
-		skreecherSpawnRolls = 1
+		skreecherSpawnRolls = 0
```

Comment:
did you test this? says range > 0 so 0 might not work

Reply:
Spawnrolls are already to 0 for many mobs like rhinoceros, sugarglider, ...

Reply:
ah okay hopefully its good then!